package/utils/validateCssModulesConfig.js

```
@@ -0,0 +1,74 @@
+const validateCssModulesConfig = (cssLoaderOptions) => {
+  if (!cssLoaderOptions || !cssLoaderOptions.modules) {
+    return
+  }
+
+  const { modules } = cssLoaderOptions
+
+  // Check for conflicting namedExport and esModule settings
+  if (modules.namedExport === true && modules.esModule === false) {
+    warn(
+      "⚠️  CSS Modules Configuration Warning:\n" +
+      "    namedExport: true with esModule: false may cause issues.\n" +
```

ESLint

[error] Insert `··`

(prettier/prettier)